packages/governance/test/swingsetTests/contractGovernor/governedContract.js

```diff
@@ -1,47 +1,43 @@
 // @ts-check
 
 import { Far } from '@agoric/marshal';
-import { E } from '@agoric/eventual-send';
 import { assert, details as X } from '@agoric/assert';
 import { sameStructure } from '@agoric/same-structure';
 
 import { buildParamManager, ParamType } from '../../../src/paramManager';
 
 const MALLEABLE_NUMBER = 'MalleableNumber';
 
-export const governedParameterTerms = {
-  contractParams: [MALLEABLE_NUMBER],
-};
+const governedParameterTerms = harden([MALLEABLE_NUMBER]);
 
-/* @type {ContractStartFn} */
+/** @type {ContractStartFn} */
 const start = async zcf => {
-  const { electionManager, governedParams } = zcf.getTerms();
-  const governorPublic = E(zcf.getZoeService()).getPublicFacet(electionManager);
+  /** @type {{ governor: Instance, governedParams: Array<string> }} */
+  const { governor, governedParams } = zcf.getTerms();
+
+  assert(
+    sameStructure(governedParams, governedParameterTerms),
+    X`governedParams must be ${governedParameterTerms}`,
+  );
+
   const paramManager = buildParamManager([
     {
       name: MALLEABLE_NUMBER,
-      value: 602214090000000000000000n,
+      value: 602214090000000000000000n, // initial value
       type: ParamType.NAT,
     },
   ]);
-  assert(
-    sameStructure(governedParams, harden(governedParameterTerms)),
-    X`Terms must include ${MALLEABLE_NUMBER}`,
-  );
-
-  const gCreator = await E(governorPublic).governContract(
-    zcf.getInstance(),
-    paramManager,
-    'contractParams',
-  );
 
   const publicFacet = Far('public face of governed contract', {
-    getState: () => paramManager.getParams().MalleableNumber,
-    getContractGovernor: () => governorPublic,
+    getParams: paramManager.getParams,
+    getGovernor: () => governor,
+    addOne: () =>
+      /** @type {NatValue} */ (paramManager.getParams()[MALLEABLE_NUMBER]
+        .value) + 1n,
   });
 
   const creatorFacet = Far('creator facet of governed contract', {
-    getContractGovernor: () => gCreator,
+    getParamManager: () => paramManager,
```

Comment:
The creator, the contract that creates this instance, would be the only one to get access to the paramManager